src/circuits/joinsplit.tcc

```diff
@@ -3,8 +3,10 @@
 
 // Contains the circuits for the notes
 #include "circuits/notes/note.hpp"
+
```

Comment:
The build still fails. Have you even read the build error?

Reply:
Thanks for telling me more in detail !